### PR TITLE
OSDOCS#5181: Add OSDK version to common attributes and clean up

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.12+

Issue: [OSDOCS-5181](https://issues.redhat.com//browse/OSDOCS-5181)

Link to docs preview:
- [Running the bundle validate command](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-bundle-validate.html#osdk-bundle-validate-run_osdk-bundle-validate) (step 2 and 3)
- [Installing the Operator SDK CLI](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-installing-cli.html) (Note admonition, step 3, and Example output)
- [Scorecard configuration](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-scorecard.html#osdk-scorecard-config_osdk-scorecard) (example config file)
- [Scorecard output](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-scorecard.html#osdk-scorecard-output_osdk-scorecard) (Example output for JSON and text)
- [Enabling parallel testing](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-scorecard.html#osdk-scorecard-parallel_osdk-scorecard) (quay.io URLs)
- [Updating Ansible projects](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-updating-projects.html)
- [Updating Go projects](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/golang/osdk-golang-updating-projects.html)
- [Updating Helm projects](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html)
- [Updating Hybrid Helm projects](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html)
- [Updating Java projects
](https://55703--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html)

QE review:
- [x] QE has approved this change.

Additional information: I added the merge review label, but I can merge and cherrypick the PR if it is approved.

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
@kalexand-rh When you have the chance, would you please ack this change to the `common-attributes.adoc` file? I think this PR might be exempt from QE verification, but would you please confirm that as well? Thank you!

**Review by adellape — APPROVED** [review]

**Review by kalexand-rh — COMMENTED** [review]
The common-attributes.adoc change LGTM.

**Comment by kalexand-rh** [comment]
@michaelryanpeter, as long as you verify that each of the commands is unchanged from the prod docs, we can count this one as a maintenance change and skip QE review.

**Comment by michaelryanpeter** [comment]
@emmajiafan Would you PTAL at this PR?

While testing some of the updated files in this PR, I noticed the downstream binary for OCP 4.12.2 is still 1.25.1: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/4.12.2/

This PR simplifies the process to update the `operator-sdk` version number in the docs and fixes the version number to 1.25.1.

**Comment by emmajiafan** [comment]
@michaelryanpeter 
 The version will be 1.25.4 since ocp4.12.3.

**Comment by michaelryanpeter** [comment]
@emmajiafan I created OSDOCS-5181 to track the change for OSDK 1.25.4/OCP 4.12.3. After this PR merges, updating the `operator-sdk` version will only require updating the `osdk_ver` variable in the `common_attributes` file.

Does that sound good to you? Are you comfortable merging this and updating the version when 4.12.3 is officially released?

cc: @mwerner2113

File: _attributes/common-attributes.adoc
Comment (emmajiafan) [inline _attributes/common-attributes.adoc:164]: Shall we update this to "1.25.4" since the following osdk_ver is 1.25.4?

**Comment by michaelryanpeter** [comment]
/label merge-review-needed

Note: I can merge and CP the PR. If I could just please have another set of eyes on this after a rebase.

**Comment by stevsmit** [comment]
This looks good to me @michaelryanpeter . Went through it twice and didn't see anything.

**Pull request merged**
